bin/post_compile

```diff
@@ -1,5 +1,12 @@
 #!/usr/bin/env bash
 
+
+# Install requirements-dev.txt on staging
+if [ "$HEROKU_APP_NAME" == "mozilla-pontoon-staging" ]; then
```

Comment:
Instead of hardcoding the instance name, maybe use `DJANGO_DEV` here and set it to True on stage?

Reply:
hmm, maybe they are independent

```
DJANGO_DEBUG:                 False
DJANGO_DEV:                   False
DJANGO_LOG_LEVEL:             DEBUG
```

ill update and see what happens...

Reply:
We might need a new variable - `STAGE`.

Reply:
@mathjazz ive updated with `DJANGO_DEV` and checked staging/logs etc - all looks good.

i think this is the right way to go - but i can change to `STAGE` if you want